tools/@aws-cdk/spec2cdk/lib/generate.ts

```diff
@@ -58,17 +63,28 @@ export interface GenerateOptions extends Pick<AstBuilderProps, 'filePatterns' |
   readonly clearOutput?: boolean;
 
   /**
-   * Generate L2 stub support files for augmentations (only for testing)
-   *
+   * Output debug messages
    * @default false
    */
-  readonly augmentationsSupport?: boolean;
```

Comment:
This was an unused option that silently got ignored. Instead augmentations were always enabled.